test/integration/dialects/mysql/connector-manager.test.js

```diff
@@ -15,19 +15,23 @@ if (Support.dialectIsMySQL()) {
         , self = this;
 
       return User.sync({force: true}).then(function() {
-        return User.create({username: 'user1'}).then(function() {
+        return User.count({group: ['id']}).then(function(count) {
+          expect(count.length).to.equal(0);
```

Comment:
This is probably related to the issue mentioned on IRC the other day @mickhansen 

When grouping count will return several rows - we haven't really decided what should happen it this case